sig-governance.md

```diff
@@ -0,0 +1,179 @@
+# SIG Roles and Organizational Governance
+
+This charter adheres to the conventions described in the [Open Data Hub Charter Guide](./sig-charter-guide.md).
+It will be updated as needed to meet the current needs of the Open Data Hub project.
+
+In order to standardize Special Interest Group efforts, create maximum
+transparency, and route contributors to the appropriate SIG, SIGs should follow
+these guidelines:
+
+- Have an approved Charter ([SIG charter process](sig-charter-guide.md))
+- Meet regularly, at least for 30 minutes every 4 weeks, except November and
+December.
+- Keep up-to-date meeting notes, linked from the SIG's page in the community
+repository.
+- Record meetings and make them publicly available.
+- Report activity with the community via the [contributors@lists.opendatahub.io](mailto:contributors@lists.opendatahub.io) mailing list at
+least once a year. 
+  - Each SIG is assigned an update during the [Open Data Hub community meeting](./README.md) throughout the year.
```

Comment:
Is this linking the right location?

Reply:
> Is this linking the right location?

`sig-governance.md` is on the same level as `README.md`